relay/nakama/allowlist_test.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/heroiclabs/nakama-common/api"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/suite"
+	"pkg.world.dev/world-engine/assert"
+	"pkg.world.dev/world-engine/relay/nakama/mocks"
+)
+
+type AllowListTestSuite struct {
+	suite.Suite
+	originalAllowListEnabled bool
+}
+
+func TestAllowList(t *testing.T) {
+	suite.Run(t, &AllowListTestSuite{})
+}
+
+func (a *AllowListTestSuite) SetupTest() {
+	a.originalAllowListEnabled = allowlistEnabled
+	allowlistEnabled = true
+}
+
+func (a *AllowListTestSuite) TearDownTest() {
+	allowlistEnabled = a.originalAllowListEnabled
+}
+
+func (a *AllowListTestSuite) TestUserIDRequired() {
+	t := a.T()
+	// This context does not have a user ID.
+	ctx := context.Background()
+
+	_, err := claimKeyRPC(ctx, noopLogger(t), nil, nil, "")
+	// claimKeyRPC should fail because the userID cannot be found in the context
+	assert.Check(t, err != nil)
```

Comment:
looks like the assert package has an `IsError` function that will do this behavior

Reply:
done